pkg/model/crd.go

```diff
@@ -315,6 +315,9 @@ func (r *CRD) UnpackAttributes() {
 // IsPrimaryARNField returns true if the supplied field name is likely the resource's
 // ARN identifier field.
 func (r *CRD) IsPrimaryARNField(fieldName string) bool {
+	if r.genCfg != nil && !r.genCfg.IncludeACKMetadata {
```

Comment:
@jaypipes since we know that it will always have some value, what would you think about making `r.genCfg` a value variable instead of pointer? We wouldn't need to make if `r.genCfg != nil` check everywhere and run into risk of panicing in places we don't.